scheduling/storage_providers/memory.go

```diff
@@ -40,30 +40,27 @@ func NewMemoryStorageProvider() *MemoryStorageProvider {
 
 func (m *MemoryStorageProvider) Set(key string, value string, lock LockObject) error {
```

Comment:
sanity check: are we sure `MemoryStorageProvider`'s storage + lockedItems shouldn't be pointers?

Reply:
These files are no longer included